docs/code/interfaces/main.c

```diff
@@ -11,17 +11,17 @@ int main() {
 
     printf("Number of interfaces: %d\n", count);
     while (i--) {
-        uv_interface_address_t interface = info[i];
+        uv_interface_address_t interface_a = info[i];
```

Comment:
For my curiosity: why the name change? Does msvc complain "interface" is a reserved word?

Reply:
Yeap. interface is a reserved word on MSVC.